### DISCUSSION TITLE
Need tip for geting responsivity from the web ui

### DISCUSSION THREAD

**Discussion by metzo007** [TheLastBen/fast-stable-diffusion#2101] [Q&A]
when I get the web interface up and running, most of the time I can only generate 2 or two pictures until the ui stops working

So I create a batch with 40 picture and use dynamic prompts. This works fine, but after the 40 pictures I need to restart the UI all the time because it takes forever to load the pictures into the UI. 

I can not find the setting to disable this. Someone knows how to do it?

Best, Metulski

**Comment by meunumerotim333** [comment]
Are you using on ngrok server? is more stable

**Comment by metzo007** [comment]
thanks. I try it
